formats/summary_test.go

```diff
@@ -1,199 +1,413 @@
 package formats
 
 import (
+	"sort"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestSummaryCount(t *testing.T) {
+func TestCuratedPackages(t *testing.T) {
 	testCases := []struct {
-		name     string
-		count    SummaryCount
-		expected int
+		name             string
+		curatedPackages  CuratedPackages
+		expectedApproved int
+		expectedBlocked  int
 	}{
-		{"Empty", SummaryCount{}, 0},
-		{"Single", SummaryCount{"High": 1}, 1},
-		{"Multiple", SummaryCount{"High": 1, "Medium": 2, "Low": 3}, 6},
+		{"Empty", CuratedPackages{}, 0, 0},
+		{"Approved", CuratedPackages{PackageCount: 1}, 1, 0},
+		{"Blocked", CuratedPackages{Blocked: []BlockedPackages{{Packages: map[string]int{"npm://test:1.0.0": 1}}}, PackageCount: 1}, 0, 1},
+		{
+			"Multiple",
+			CuratedPackages{
+				Blocked: []BlockedPackages{
+					{
+						Policy:    "Test",
+						Condition: "Test condition",
```

Comment:
can we have more descriptive texts here?
I saw some examples of "aged packages", "immature", "highProfile", same note in condition "cvss > 7.5"

Reply:
you have realistic in `securityJobSummary_test` where you can actually view the UI that is generated.
This test is only for logic of counting